test/built-ins/Array/prototype/concat/concat-array-likes-1.js

```diff
@@ -0,0 +1,677 @@
+// Copyright 2014 the V8 project authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flags: --harmony-arrays --harmony-classes
+
+/*---
+es6id: 22.1.3.1_3
+description: >
+    Array.prototype.concat: Symbol.isConcatSpreadable & array-likes
+---*/
+
+(function testArrayConcatArity() {
+  "use strict";
+  assert.deepEquals(1, Array.prototype.concat.length);
```

Comment:
Actually, this was from last week, before we discovered all of the unnecessary and ambiguous uses of `assertEquals` and had a transformer that automatically changed all `assertEquals` to `assert.deepEquals` (as illustrated in the table [here](https://github.com/tc39/test262/pull/149#issuecomment-73975846)).